config/config.json

```diff
@@ -19,13 +19,10 @@
 	},
 	"DBConf": {
 		"Protocol": "tcp",
-		"Host": "localhost",
-		"Port": "6380",
+		"Address": "localhost:6380",
 		"MinIdleConns": 10,
 		"PoolSize": 120,
-		"RedisHAEnabled": false,
-		"SentinelPort": "26379",
-		"MasterSet": "mymaster"
+		"MasterName": "mymaster"
```

Comment:
How do we now know whether HA is enabled or not?

Reply:
How do we know the different cluster nodes? Do we need to ?

Reply:
reverted the implementation for HA